pihole-updatelists.php

```diff
@@ -1401,7 +1401,7 @@ function textToArray($text)
 
     foreach ($array as $var => &$val) {
         // Ignore empty lines and those with only a comment
-        if (empty($val) || strpos(trim($val), '#') === 0) {
+        if (empty($val) || strpos(trim($val), '#') === 0 || strpos(trim($val), '=') === 0) {
```

Comment:
Getting a bit confused here, why ignore '='? I don't think any line should be starting with it anyway

Reply:
I added it because [this list](https://raw.githubusercontent.com/Sekhan/TheGreatWall/master/TheGreatWall.txt) uses a line of equal signs to separate the entries from the comments.